srcpkgs/v2ray/files/v2ray/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec chpst -u nobody:nogroup v2ray -config=/etc/v2ray/config.json 2>&1
```

Comment:
```suggestion
exec chpst -u v2ray:v2ray v2ray -config=/etc/v2ray/config.json 2>&1
```

Reply:
Accounts provided by packages should be prefixed with `_`: `_v2ray`.